test/e2e/commands/separate_node_add_local.test.mjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the ""License"");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an ""AS IS"" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @jest-environment steps
+ */
+import { describe } from '@jest/globals'
+import { testNodeAddSeparated } from '../../test_add_separate.mjs'
+
+describe('Node add via separated commands with hedera local build', () => {
```

Comment:
I would recommend just removing the test run against the local build, it is tested enough in other ways.  In my PR, I'm splitting these all out into separate tests in the pipeline, also.  So, if you want to keep it, then I would make it a separate job in the pipeline instead of running two of them in the same job.